PorticoExportDom.inc.php

```diff
@@ -117,12 +118,23 @@ private function _buildArticle() : DOMElement {
 		$articleNode->appendChild($articleMetaNode);
 
 		// article-id (DOI)
-		if (($doi = $article->getStoredPubId('doi')) != '') {
+		if (($doi = $article->getStoredPubId('doi'))) {
 			$doiNode = $doc->createElement('article-id', $doi);
 			$doiNode->setAttribute('pub-id-type', 'doi');
 			$articleMetaNode->appendChild($doiNode);
 		}
 
+		// article-id (PII)
+		// Pubmed will accept two types of article identifier: pii and doi
+		// how this is handled is journal-specific, and will require either
+		// configuration in the plugin, or an update to the core code.
+		// this is also related to DOI-handling within OJS
+		if ($publisherId = $article->getStoredPubId('publisher-id')) {
```

Comment:
I've moved this up, since the elements are ordered, and also removed the `<ArticleIdList>` tag as it's not in the DTD.

Reply:
:+1: